RecoTauTag/Configuration/python/tools/helpers.py

```diff
@@ -0,0 +1,72 @@
+import FWCore.ParameterSet.Config as cms
+
+## Helpers to perform some technically boring tasks like looking for all modules with a given parameter
+## and replacing that to a given value
+
+# Next two lines are for backward compatibility, the imported functions and
+# classes used to be defined in this file.
+from FWCore.ParameterSet.MassReplace import massSearchReplaceAnyInputTag, MassSearchReplaceAnyInputTagVisitor
+from FWCore.ParameterSet.MassReplace import massSearchReplaceParam, MassSearchParamVisitor, MassSearchReplaceParamVisitor
+
+class CloneTaskVisitor(object):
```

Comment:
this looks like an extension to what's available in `PhysicsTools.PatAlgos.tools.helpers.CloneSequenceVisitor`
I think that this part is somewhat misplaced.
The simpler change is perhaps to add these generic helpers to `PhysicsTools.PatAlgos.tools.helpers`,
but eventually, some place in `FWCore.ParameterSet` would be more appropriate.

@cms-sw/core-l2 please check and advise if placement in `FWCore.ParameterSet` is reasonable
Thank you.

Reply:
To me (e.g.) the `PhysicsTools.PatAlgos` (continues to) looks a reasonable place for these helpers.

Reply:
Thanks for the advice, the helper will be moved.